types/luxon/src/datetime.d.ts

```diff
@@ -406,6 +406,86 @@ export interface TokenParser {
+ * | kk               |              | ISO week year, unpadded                                           | `17`                        |
+ * | kkkk             |              | ISO week year, padded to 4                                        | `2014`                      |
+ * | W                |              | ISO week number, unpadded                                         | `32`                        |
+ * | WW               |              | ISO week number, padded to 2                                      | `32`                        |
+ * | o                |              | ordinal (day of year), unpadded                                   | `218`                       |
+ * | ooo              |              | ordinal (day of year), padded to 3                                | `218`                       |
+ * | q                |              | quarter, no padding                                               | `3`                         |
+ * | D                |              | localized numeric date                                            | `9/6/2014`                  |
+ * | DD               |              | localized date with abbreviated month                             | `Aug 6, 2014`               |
+ * | DDD              |              | localized date with full month                                    | `August 6, 2014`            |
+ * | DDDD             |              | localized date with full month and weekday                        | `Wednesday, August 6, 2014` |
+ * | t                |              | localized time                                                    | `1:07 AM`                   |
+ * | tt               |              | localized time with seconds                                       | `1:07:04 PM`                |
+ * | T                |              | localized 24-hour time                                            | `13:07`                     |
+ * | TT               |              | localized 24-hour time with seconds                               | `13:07:04`                  |
+ * | f                |              | short localized date and time                                     | `8/6/2014, 1:07 PM`         |
+ * | ff               |              | less short localized date and time                                | `Aug 6, 2014, 1:07 PM`      |
+ * | F                |              | short localized date and time with seconds                        | `8/6/2014, 1:07:04 PM`      |
+ * | FF               |              | less short localized date and time with seconds                   | `Aug 6, 2014, 1:07:04 PM`   |
+ * | '                |              | literal start/end, characters between are not tokenized           | `'T'`                       |
+ *
+ * Sourced from [here](https://moment.github.io/luxon/#/parsing?id=table-of-tokens).
+ */
+type Tokens = string;
```

Comment:
I'm not 100% certain this will do what you hope; I think TS may eagerly resolve references to this to `string` and thus hide the JSDoc, but I haven't tested.

Reply:
I think it does yes. But it still lets the jsdoc `{@link}` to it. So still only a couple clicks away. Not trying to change types here, only provide docs.